plugins/source/oracledb/client/client.go

```diff
@@ -3,23 +3,22 @@ package client
 import (
 	"context"
 	"database/sql"
+	"encoding/json"
 	"fmt"
 
-	"github.com/cloudquery/plugin-pb-go/specs"
-	"github.com/cloudquery/plugin-sdk/v3/plugins/source"
-	"github.com/cloudquery/plugin-sdk/v3/schema"
+	"github.com/cloudquery/plugin-sdk/v4/plugin"
+	"github.com/cloudquery/plugin-sdk/v4/schema"
 	"github.com/rs/zerolog"
 
 	// Import the oracle driver
 	_ "github.com/sijms/go-ora/v2"
 )
 
 type Client struct {
-	logger      zerolog.Logger
-	metrics     *source.Metrics
-	Tables      schema.Tables
-	db          *sql.DB
-	Concurrency uint64
```

Comment:
We use to read this from the top level spec, so this could be considered a breaking change as now we don't account for it

Reply:
https://github.com/cloudquery/cloudquery/pull/11766 should handle this